osquery/tables/system/windows/tests/startup_tests.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <osquery/sql.h>
+
+#include "osquery/tests/test_util.h"
+
+namespace osquery {
+namespace tables {
+
+class StartupTablesTest : public testing::Test {};
+
+TEST_F(StartupTablesTest, test_startup_table) {
+  SQL results("SELECT * FROM startup");
```

Comment:
We should get this for free with a python script that runs all of the spec's example queries.